private.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2017 SUSE LLC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: v1
+kind: Pod
+metadata:
+  name: velum-private
+  labels:
+    name: velum-private
+spec:
+  hostNetwork: false
+  containers:
+  - name: velum-mariadb
+    image: sles12/mariadb:10.0
+    env:
+    - name: MYSQL_ROOT_PASSWORD
+      value: "salt"
```

Comment:
I created https://trello.com/c/VB40KK1s/211-generate-infrastructure-secrets, feel free to expand or improve it as you see fits :)